config/environments/test.rb

```diff
@@ -39,6 +39,8 @@
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
 
+  config.active_job.queue_adapter = :test
```

Comment:
Hmm, I guess moving it here is good, I'm not sure if we were using other delayed job adapters elsewhere.

Reply:
It seems like the tests are all still passing in CI with this change. I didn't find any other places where we are setting the queue_adaptor.